jarjar/src/main/java/com/eed3si9n/jarjar/util/EntryStruct.java

```diff
@@ -21,4 +21,17 @@ public class EntryStruct {
     public String name;
     public long time;
     public boolean skipTransform;
+
+    public EntryStruct copy() {
+        EntryStruct result = new EntryStruct();
+        result.assign(this);
+        return result;
+    }
+
+    public void assign(EntryStruct other) {
+        data = other.data;
+        name = other.name;
+        time = other.time;
+        skipTransform = other.skipTransform;
+    }
```

Comment:
this public method is no longer necessary
```suggestion
        data = other.data;
        name = other.name;
        time = other.time;
        skipTransform = other.skipTransform;
        return result;
    }
```